infrastructure/tooling/src/dev/taskcluster.js

```diff
@@ -51,6 +51,8 @@ module.exports = async ({ userConfig, answer, configTmpl }) => {
   // Simple defaults that can be updated by the user later
   setDefault('worker_manager.providers', {});
   setDefault('web_server.public_url', rootUrl);
+  setDefault('web_server.auth_rate_limit_max_requests', 5);
```

Comment:
this feels like a very low limit, but I don't know what clients are using this to authorize, so might be good enough